rom/laghos.cpp

```diff
@@ -1310,7 +1349,7 @@ int main(int argc, char *argv[])
                 for (int curr_window = numWindows-1; curr_window >= 0; --curr_window)
                     basis[curr_window]->writePDweights(pd2_tdof, curr_window);
             }
-            if (!romOptions.hyperreduce_prep)
+            if (romOptions.hyperreduce)
```

Comment:
If I remember correctly, we intentionally leave the option of online ROM via Galerkin projection but **NOT** using hyper-reduction, which I believe is because we want to see the fact that hyper-reduction is needed for speed-up. But we ended up having these strange conditions `!romOptions.hyperreduce_prep`. Should we get rid of this consideration from now on?

Reply:
This change looks like a mistake. It has nothing to do with this PR. I will revert this change. You still have a valid question, which I think should be addressed outside this PR.